Tasks/CopyFilesV2/copyfiles.ts

```diff
@@ -107,6 +108,9 @@ if (matchedFiles.length > 0) {
                 else { // copy
                     console.log(tl.loc('CopyingTo', file, targetPath));
                     tl.cp(file, targetPath);
+                    if (keepTimestamp) {
+                        fs.utimes(targetPath, targetStats.atime, targetStats.mtime);
```

Comment:
Looks like you need to include a callback parameter to avoid a warning on node >=7 and an error on node 10. From https://nodejs.org/api/fs.html#fs_fs_utimes_path_atime_mtime_callback - ```The callback parameter is no longer optional. Not passing it will throw a TypeError at runtime.```

@juliobbv FYI, this is the type of thing we need to watch out for as we move to Node 10, looks like back-compat isn't 100% there. Just from a cursory view I counted 30 different functions in the fs module that gave deprecation warnings in node 7 that now throw errors (and I think didn't even have warnings in node 6).

Reply:
Additionally, at this point in the code targetStats hasn't been initialized (we're in the else clause of ```if (targetStats)```, so this will blow up. Additionally, targetStats was initially set to get stats about the targetPath, not the path we're copying from. I think what you want is to get ```tl.stats(file)```

Reply:
I think this has been resolved? Line 112 `tl.stats(file);` is used to initialize fileStats to be used.